tests/src/test/java/alluxio/client/cli/fs/command/LoadMetadataCommandV2IntegrationTest.java

```diff
@@ -0,0 +1,247 @@
+  protected LocalAlluxioJobCluster mLocalAlluxioJobCluster;
+  protected JobShell mJobShell;
+
+  @Rule
+  public LocalAlluxioClusterResource mLocalAlluxioClusterResource = new LocalAlluxioClusterResource.Builder()
+            .setProperty(PropertyKey.USER_FILE_BUFFER_BYTES, USER_QUOTA_UNIT_BYTES)
+            .setProperty(PropertyKey.UNDERFS_S3_ENDPOINT, "localhost:8001")
+            .setProperty(PropertyKey.UNDERFS_S3_ENDPOINT_REGION, "us-west-2")
+            .setProperty(PropertyKey.UNDERFS_S3_DISABLE_DNS_BUCKETS, true)
+            .setProperty(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, "s3://" + TEST_BUCKET)
+            .setProperty(PropertyKey.S3A_ACCESS_KEY, mS3Proxy.getAccessKey())
+            .setProperty(PropertyKey.S3A_SECRET_KEY, mS3Proxy.getSecretKey())
+            .setStartCluster(false)
+            .build();
+
+  private AmazonS3 mS3Client = null;
+
+
+
+  @Before
+  public void before() throws Exception {
+    mS3Client = AmazonS3ClientBuilder
+        .standard()
+        .withPathStyleAccessEnabled(true)
+        .withCredentials(
+            new AWSStaticCredentialsProvider(
+                new BasicAWSCredentials(mS3Proxy.getAccessKey(), mS3Proxy.getSecretKey())))
+        .withEndpointConfiguration(
+            new AwsClientBuilder.EndpointConfiguration(mS3Proxy.getUri().toString(),
+                Regions.US_WEST_2.getName()))
+        .build();
+    mS3Client.createBucket(TEST_BUCKET);
+
+    System.out.println(mS3Client.listBuckets());
+
+    mLocalAlluxioClusterResource.start();
+    mLocalAlluxioCluster = mLocalAlluxioClusterResource.get();
+    mLocalAlluxioJobCluster = new LocalAlluxioJobCluster();
+    mLocalAlluxioJobCluster.start();
+    mFileSystem = mLocalAlluxioCluster.getClient();
+    mJobMaster = mLocalAlluxioJobCluster.getMaster().getJobMaster();
+    mJobShell = new alluxio.cli.job.JobShell(Configuration.global());
+    mFsShell = new FileSystemShell(Configuration.global());
+  }
+
+  @After
+  public void after() throws Exception {
+    mS3Client = null;
+    if (mFsShell != null) {
+      mFsShell.close();
+    }
+    if (mLocalAlluxioJobCluster != null) {
+      mLocalAlluxioJobCluster.stop();
+    }
+    if (mJobShell != null) {
+      mJobShell.close();
+    }
+  }
+
+  @Test
+  public void test() {
+    String str = "...... Task group 0 has been submitted successfully. ......";
+    Pattern pattern = Pattern.compile("Task group (\\d+)");
+    Matcher matcher = pattern.matcher(str);
+    if (matcher.find()) {
+      String match = matcher.group(1); // 获取匹配到的数字字符串
+      int num = Integer.parseInt(match); // 将字符串转换为 int 类型
+      System.out.println(num);
+    }
+  }
+
+  // The main idea of this test is start an async loadMetadata task and get its status
+  // Totally three status were tested, RUNNING, CANCELED, SUCCESSES
+  @Test
+  public void loadMetadataTestV2get() throws IOException, AlluxioException {
+    // the cancel dir should be big enough
+    for (int i = 0; i < 100; i++) {
```

Comment:
instead of creating 100 files which we "believe" is big enough. Can we try using a SleepingUnderFileSystemFactory? Check examples in LoadMetadataIntegrationTest